cypress/support/commands/application.js

```diff
@@ -31,12 +31,13 @@ Cypress.Commands.add(
 
             // Wait before retrying to allow for transient problems to go away
             // eslint-disable-next-line cypress/no-unnecessary-waiting
-            cy.wait(5000).testResponseCodeAndType(
+            cy.wait(5000).testResponseCodeAndType({
               path,
               responseCode,
               type,
-              retriesLeft - 1,
-            );
+              retriesLeft: retriesLeft - 1,
+              allowFallback,
```

Comment:
```suggestion
              allowFallback: false,
```
In this code path, `allowFallback` is always false, maybe set this explicitly here as it confused me a little that it could somehow be true here?